website/learn-programming-challenge/basic-game-rules/game-rules-deep-dive.md

```diff
@@ -72,18 +73,17 @@ Each planet occupies a perfectly circular area. Planets have:
 ### Docking and Mining
 To take control of a planet, a player must dock a ship to the planet. Only one team can dock on a planet at a time.
 
-**Docking:** Once a ship moves within 4 units of a planet (ie. 4 + radius distance from center of planet), it can be commanded to dock, which will cause the ship to begin the docking process. The planet must be unoccupied or owned by player currently intending to dock. The ship must also be stationary. If the ship is too far away or not stationary or the planet is occupied, the dock command does nothing.
+**Docking:** Once a ship moves within 4 units of a planet (ie. 4 + radius distance from center of planet), it can be commanded to dock, which will cause the ship to begin the docking process. The planet must be unoccupied or owned by the player intending to dock. The ship must also be stationary. If the ship is too far away or not stationary or the planet is occupied, the dock command does nothing.
 
 If two ships from different teams both issue docking commands on an unoccupied planet during the same turn, they will battle. If they both continue to issue docking commands, they will continue to battle. During these turns, the ships are not yet docking and therefore maintain their defenses (described under docking section below) until they start to successfully dock.
 
-Once a planet has been docked by one ship, the owning player may continue to dock ships to the planet until the limit of ships per that planet has been reached. The maximum number of ships that can be docked to a planet is a function of the radius. E.g. a planet with a __radius of 3 units may have up to 3 ships__ docked to it at once.
-
-If a ship attempts to dock on a full side of a planet (all spots full) the docking process will move them to the closest unoccupied spot on the planet until the planet is full. If the planet is full, the dock command does nothing.
+Once a player starts to dock, that player is considered the owner of the planet. The owning player may continue to dock ships to the planet until the limit of ships per that planet has been reached. The maximum number of ships that can be docked to a planet is a function of the radius: larger planets have more docking spots.
```

Comment:
Might want to mention here that for the purpose of ending the game, the player is not considered to own the planet until docking completes.

Reply:
Fixed